<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="button1">
        <property name="text">
         <string>Load File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button2">
        <property name="text">
         <string>Execute File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button3">
        <property name="text">
         <string>Save As</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button4">
        <property name="text">
         <string>Clear console</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="appLayout">
      <item>
       <layout class="QVBoxLayout" name="consoleLayout">
        <item>
         <widget class="QLabel" name="consoleLabel">
          <property name="text">
           <string>Console</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="console"/>
        </item>
        <item>
         <widget class="QLineEdit" name="consoleField"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="editorLayout">
        <item>
         <widget class="QLabel" name="editorLabel">
          <property name="text">
           <string>Editor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textViewer"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
